templates/repo/editor/patch.tmpl

```diff
@@ -45,14 +45,14 @@
 			<p>{{.locale.Tr "repo.editor.commit_empty_file_text"}}</p>
 		</div>
 		<div class="actions">
-			<div class="ui red basic cancel inverted button">
-				<i class="remove icon"></i>
+			<button class="ui red basic inverted cancel button">
+				{{svg "octicon-x"}}
 				{{.locale.Tr "repo.editor.cancel"}}
-			</div>
-			<div class="ui green basic ok inverted button">
-				<i class="save icon"></i>
+			</button>
+			<button class="ui green basic inverted ok button">
+				<i class="save icon"></i>{{/* TODO: Is there no fitting octicon? */}}
```

Comment:
Could just port the fontawesome icon:

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="512" height="512" viewBox="0 0 448 512"><path d="m434 130-84-84a48 48 0 0 0-33.9-14H48A48 48 0 0 0 0 80v352a48 48 0 0 0 48 48h352a48 48 0 0 0 48-48V163.9a48 48 0 0 0-14-34zM224 416a64 64 0 1 1 0-128 64 64 0 0 1 0 128zm96-304.5V212a12 12 0 0 1-12 12H76a12 12 0 0 1-12-12V108a12 12 0 0 1 12-12h228.5a12 12 0 0 1 8.5 3.5l3.5 3.5a12 12 0 0 1 3.5 8.5z"/></svg>
```

Save as `web_src/svg/fontawesome-save.svg` and run `make svg`.